hypothesis-python/setup.py

```diff
@@ -19,7 +19,7 @@
 
 import setuptools
 
-if sys.version_info[:2] < (3, 5):
+if sys.version_info[:3] < (3, 5, 2):
```

Comment:
```suggestion
if sys.version_info < (3, 5, 2):
```
for consistency with `_error_if_old.py`?

Reply:
For the avoidance of doubt (comparisons between different-length tuples are tricky, and I always have to look up which are safe...) I've made the *other* usage consistent with this one :joy: